dio/lib/src/adapter.dart

```diff
@@ -4,6 +4,7 @@ import 'dart:typed_data';
 import 'package:meta/meta.dart';
 
 import 'adapters/io_adapter.dart'
+    if (dart.library.js_util) 'adapters/browser_adapter.dart'
     if (dart.library.html) 'adapters/browser_adapter.dart' as adapter;
```

Comment:
This line should be removed. ALL references to `dart:html` and `dart.library.html` should be removed.

Reply:
Actually removing these might require upping the min Dart version to 3.3 which isn't ideal

Reply:
> This line should be removed. ALL references to `dart:html` and `dart.library.html` should be removed.

Will this cause WASM to degrade or something related?

Reply:
I'm not sure. All I know is the migration docs say to remove them.

Reply:
Since this is an import condition it shouldn't cause any issues, but the js_util ones definitely need changed to js_interop as js_util is one of the deprecated web packages

Reply:
I think the imports are resolved at compile time, so it should be fine. But definitely should switch to the new `js_interop` package. We need to figure out if it is possible to run our adapter test in WASM, that will tell us that it is working